### DISCUSSION TITLE
Hide a task temporarily?

### DISCUSSION THREAD

**Discussion by xingheng** [GothenburgBitFactory/taskwarrior#3259] [Q&A] [answered]
I want to hide some tasks which already been created sometimes, they may be done someday but I have no ideas about the date currently. So I'm looking for an operation to hide them in the default `task` command list.

Are there any alternative operations for that or what should I do?

**Comment by djmitche** [comment, marked as answer]
I typically use the wait property to make the task wait until I want to think about it again. Maybe that's a week or a month or something. And maybe when that time is up I make it wait again.
